### DISCUSSION TITLE
API Control

### DISCUSSION THREAD

**Discussion by AlbertRamstein** [community/community#36593] [Apps, API and Webhooks] [answered]
What is the best liescene to have when it's come's to API Keygenerator system ?

**Comment by BLKKKBVSIK** [comment, marked as answer]
It really depends on one what right you want to give to the people who will use your project.
You can learn more at https://choosealicense.com/ or directly on the documentation 
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/licensing-a-repository

**Comment by ossanna16** [comment]
Hey there @AlbertRamstein and thank you for asking a good question! I'm glad you found our community 🙂
